package.json

```diff
@@ -12,14 +12,16 @@
     "migrate:evm": "truffle migrate",
     "migrate:ovm": "truffle migrate --skip-dry-run --config truffle-config.ovm.js --network $npm_config_network",
     "test:evm": "truffle test",
-    "test:ovm": "truffle test --config truffle-config.ovm.js --network $npm_config_network"
+    "test:ovm": "truffle test --config truffle-config.ovm.js --network $npm_config_network",
+    "exec:evm": "truffle exec",
```

Comment:
I'm a little confused why this is needed -- you can just run `truffle exec` and it will default to the EVM config. If you do need it for something specific, then we should add something about it in the readme.

Reply:
I didn't think it was necessary, but I just added to keep parity with the other commands (i.e. `migrate:evm`, `test:evm`)

Do you think it's better to just remove all the non ovm commands?

Reply:
I only made the `:evm` commands for those commands that have an `:ovm` equivalent. Does that make sense?

Reply:
There's an `exec:ovm` equivalent on line 17? I am confused lol

Reply:
Sorry, I mean to say that I didn't make every possible Truffle command this way. I didn't do the `truffle exec` one because it seemed unnecessary for the beginner flow I outlined in the readme. However, if you feel that it is necessary we can keep it in! 

Let's just add a line in the readme about the availability of this command and what it does (allows you to execute a script that has access to your Truffle configuration and a web3 instance that is connected to the network indicated).

Reply:
done!

Reply:
@fainashalts are we good to merge here?

Reply:
I made a suggestion! Otherwise looks good!